init-ci.ps1

```diff
@@ -80,6 +81,24 @@ if ([string]::IsNullOrEmpty($DemoTeamRegistry)) {
     throw
   }
```

Comment:
Please remove the retry. If DemoTeamRegistry is not passed or not found in the environment, simply output a warning.